### ISSUE TITLE
Hot Reload/Restart Multiple Targets Simultaneously 

### ISSUE THREAD

**Issue by crimsonvspurple** [flutter/flutter-intellij#6334]
## Steps to Reproduce

- Run a flutter project on web
- Run the _same_ project on emulator
- Make any changes (ctrl + s) so hot reload/restart triggers automatically

### Expected
- Since both are same project; both should hot reload at the same time (or an option to do so)

### Observed
- Only one is hot reloaded.
- It depends on which `run` tab you have last selected.
- Selecting some tab which is not of this project (so none is selected), still triggers hot reload on the last selected one.

While working on layouts that should look good on mobile and desktop, being able to hot-reload both would be _really_ nice.

## Version info

```
[√] Flutter (Channel stable, 3.0.5, on Microsoft Windows [Version 10.0.22000.978], locale en-US)
    • Flutter version 3.0.5 at C:\Users\riasa\.flutter-sdk\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision f1875d570e (3 months ago), 2022-07-13 11:24:16 -0700
    • Engine revision e85ea0e79c
    • Dart version 2.17.6
    • DevTools version 2.12.2

[!] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at C:\Users\riasa\AppData\Local\Android\sdk
    X cmdline-tools component is missing
      Run `path/to/sdkmanager --install "cmdline-tools;latest"`
      See https://developer.android.com/studio/command-line for more details.
    X Android license status unknown.
      Run `flutter doctor --android-licenses` to accept the SDK licenses.
      See https://flutter.dev/docs/get-started/install/windows#android-setup for more details.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[√] Visual Studio - develop for Windows (Visual Studio Professional 2022 17.0.0 Preview 1.1)
    • Visual Studio at C:\Program Files\Microsoft Visual Studio\2022\Preview
    • Visual Studio Professional 2022 version 17.0.31423.177
    • The current Visual Studio installation is a pre-release version. It may not be supported by Flutter yet.
    • Windows 10 SDK version 10.0.22000.0

[!] Android Studio (not installed)
    • Android Studio not found; download from https://developer.android.com/studio/index.html
      (or visit https://flutter.dev/docs/get-started/install/windows#android-setup for detailed instructions).

[√] IntelliJ IDEA Ultimate Edition (version 2022.2)
    • IntelliJ at C:\Users\riasa\AppData\Local\JetBrains\Toolbox\apps\IDEA-U\ch-0\222.4167.29
    • Flutter plugin version 70.0.5
    • Dart plugin version 222.4167.21

[√] VS Code (version 1.71.2)
    • VS Code at C:\Users\riasa\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension can be installed from:
       https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter

[√] Connected device (4 available)
    • sdk gphone64 x86 64 (mobile) • emulator-5554 • android-x64    • Android 13 (API 33) (emulator)
    • Windows (desktop)            • windows       • windows-x64    • Microsoft Windows [Version 10.0.22000.978]
    • Chrome (web)                 • chrome        • web-javascript • Google Chrome 105.0.5195.127
    • Edge (web)                   • edge          • web-javascript • Microsoft Edge 105.0.1343.50

[√] HTTP Host Availability
    • All required HTTP hosts are available

! Doctor found issues in 2 categories.
```

**Comment by stevemessick** [comment]
I'm sorry to say this isn't planned. It was implemented this way in a pre-1.0 version of the plugin, but that code was removed. (I'm not sure why.)

**Comment by crimsonvspurple** [comment]
Possible to know the reason for removal?
